### ISSUE TITLE
no examples with custom functions

### ISSUE THREAD

**Issue by yury-kozlov** [MicrosoftDocs/PowerShell-Docs#4673]
There's no example with ScriptBlock containing custom function that I want to pass to a new job

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1b1a3836-76b5-58af-e342-1da76840bf04
* Version Independent ID: 4cd839bc-6755-c611-5961-3d971255f931
* Content: [Start-Job (Microsoft.PowerShell.Core)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/start-job?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Core/Start-Job.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/Start-Job.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

**Comment by sdwheeler** [comment]
@yury-kozlov Have you tried putting your function in the script block? What problem are you having?

**Comment by yury-kozlov** [comment]
I have multiple functions defined in a single file. I tried to wrap the main entry point with a script block and start a new job with this block. Apparently this requires all referenced functions to be defined as a script block too.

I ended up using a Start-Job with -FilePath flag instead of a -ScriptBlock. 
So one script file is sync code, and another file is just a trigger that starts it async by file path.
